docs/usage_guide.rst

```diff
@@ -463,6 +463,25 @@ In case of an exchange providing a more granular permissions scheme (e.g. Coinba
 
 You may as well try creating an API key with the minimum read-related permissions, then adding it in rotki and finally checking that the connection was successful and data was loaded as expected. Otherwise, try again adding more read-related premissions.
 
+Binance / Binance US
+--------------
+
+Binance API is engineered in a way that makes really slow to query information for trades since every possible market pair has to be queried. This process can also fail since many requests have to be made to binance's servers and some rate limits could apply.
+To avoid having to query every possible trade pair, since version 1.18, we have made possible to select what markets should be queried. This will considerably increase the speed as the amount of queries to binance will be reduced to only the markets you specify.
```

Comment:
```suggestion
To avoid having to query all existing trade pairs, it is possible to select what markets should be queried. This will considerably increase the speed as the amount of queries to binance will be reduced to only the markets you specify.
```